run

```diff
@@ -1,5 +1,19 @@
 #!/bin/bash
-pushd $(dirname $0) >/dev/null
-data_dir="$(pwd)/data"
-docker run -ti --rm --net host -v "$data_dir":/data networkboot/dhcpd "$@"
-popd >/dev/null
+
+MACHINE_TYPE=`uname -m`
+
+if [ "${MACHINE_TYPE}" = 'x86_64' ]; then
+  MACHINE_TAG=
+elif [ "${MACHINE_TYPE}" = armv7l ]; then
+  MACHINE_TAG=':arm32v7'
+else
+  echo "Arhictecture not supported: $MACHINE_TYPE" 1>&2
+  exit 1
+fi
+
+if [ -z "$DATA_DIR" ]; then
+  DATA_DIR="$(pwd)/data"
```

Comment:
You need to add the pushd/popd commands around this command to ensure that the previous behavior still works without changs (which is that the data directory next to the run script is used, even if the run script is started when current directory is somewhere else).

Reply:
Sorry, I overlooked the directory change.

By the way, the effect of `pushd` is lost outside a script, because `cd` doesn't affect the parent process. For example:

```console
$ cd /tmp
$ echo 'pushd ~' > foo.sh
$ chmod +x foo.xh
$ ./foo.sh
~ /tmp
$ pwd
/tmp
$
```

So I think a simple `cd` will suffice.

Reply:
The `cd` command must be _before_ the setting of `DATA_DIR`, to ensure `pwd` returns the expected directory.